keyboards/handwired/dactyl_manuform/5x6/keymaps/squirrel/config.h

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2012 Jun Wako <wakojun@gmail.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+
+#pragma once
+
+
+#define USE_SERIAL
+
+#define MASTER_LEFT
+// #define MASTER_RIGHT
+//#define EE_HANDS
+
+// Mouse
+#undef MOUSEKEY_INTERVAL
+#define MOUSEKEY_INTERVAL       15
+#undef MOUSEKEY_TIME_TO_MAX
+#define MOUSEKEY_TIME_TO_MAX    60
+#undef MOUSEKEY_WHEEL_MAX_SPEED
+#define MOUSEKEY_WHEEL_MAX_SPEED   	3
+
+// RGB backlight
+#undef RGB_DI_PIN
+#define RGB_DI_PIN D3
+#undef RGBLED_NUM
+#define RGBLED_NUM 30
+#define RGBLIGHT_ANIMATIONS
+#define RGBLIGHT_LIMIT_VAL 110
+#define RGBLIGHT_SPLIT
```

Comment:
Do you have 30 LEDs total?  Or 30 per side?

Reply:
30 per side.  it's working, i tested it out.  https://www.youtube.com/watch?v=knwKASuZ8so

Reply:
Ah, okay.  And cool. 

Though ... what may work better for you, is :
```c
#define RGBLED_NUM 60
#define RGBLIGHT_SPLIT
#define RGBLED_SPLIT { 30, 30}
```

This may get the a similar effect, it should handle things better. 

Also, if the LEDs are placed under each key (I suspect that they're not), then you could use the rgb matrix feature, instead.

Reply:
@drashna I gave it a try, and honestly the colors aren't as balanced between the two hands with this setting.  i'm going to keep my keyboard with the way i had it.  is there a future compatibility issue with the current setup, in that it just-so happens to currently work but might go wild if i would update my fork in the future with this type of config?  I can change it to be the way you have it, if you insist on it for the main repo, though I'm currently planning to keep my keyboard the previous way, with the rgbled_num and no explicit led_split designation

Reply:
No worries!  And if it works ... it works.